frontend/src/components/resourceQuota/Details.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import ResourceQuota from '../../lib/k8s/resourceQuota';
+import { DetailsGrid, SimpleTable } from '../common';
+
+export default function ResourceQuotaDetails() {
+  const { namespace, name } = useParams<{ namespace: string; name: string }>();
+  const { t } = useTranslation();
+
+  return (
+    <DetailsGrid
+      resourceType={ResourceQuota}
+      name={name}
+      namespace={namespace}
+      extraInfo={item =>
+        item && [
+          {
+            name: t('frequent|Status'),
+            value: (
+              <SimpleTable
+                data={item.resourceStats}
+                columns={[
+                  {
+                    label: t('resourceQuota|Resource'),
+                    getter: item => item.name,
+                  },
+                  {
+                    label: t('resourceQuota|Used'),
+                    getter: item => item.used,
+                  },
+                  {
```

Comment:
I think we may have these and other labels already translated to the same or very similar meaning elsewhere. If that's the case, we should reuse those translations rather than creating new ones for this new namespace.

Reply:
I reverified, `Resources` exists in the `glossary` but this is `Resource`.